### ISSUE TITLE
Still about standardization

### ISSUE THREAD

**Issue by khan1792** [UC-MACSS/persp-analysis#481]
I saw the last issue and the answer about standardization. Although you do not recommend standardization in this homework, I have a different opinion about it and standardize variables in the homework.

When I tried both standardization and non-standardization in PCA, the results were totally different. For example, in the first part, the cumulative PVE of the first two components are 58.4% and 87% respectively in standardization and non-standardization. Besides, under these two circumstances, the role of variables in the first two principle components are also totally different. 

PCA computes its components through orthogonal projection of the data onto a lower dimensional linear space. If we don't standardize variables, the angle of dimensional linear space will be based more on large-scale variables in order to minimize total errors and maximize total variances.

In the first part, there are 9 large-scale variables and 9 small-scale variables. If we don't standardize them, the result based on 18 variables will be very similar to that just based on 9 large-scale variables among them. However, if we standardize them, they will be very different. In this case, in the first part, the first two components can explain different percentages of variances under two circumstances in the same data set because small-scale variables do matter when we standardize them. In the same logic, the first 5 graphs of the second part of the homework would also look different under two circumstances.   

Probably in this homework both methods are correct, but I just think standardization is necessary in most real data analysis. In addition, ISLR also recommends us to scale variables (p. 381).

**Comment by ZhengErWei** [comment]
I cannot agree more. In ISLR it also says standardization is important to PCA though sometimes not necessary for clustering.

**Comment by bensoltoff** [comment]
Standardization is a common feature transformation in certain statistical learning models. As you correctly point out, under certain methods using standardized vs. non-standardized variables will lead to drastically different outcomes. This is not just a feature of PCA, but other methods of analysis as well. I expected it to be a topic of focus next term in the modeling course, so I did not focus on it here.

That said I just double-checked my answer key and I did use standardized variables for the model (depending on R/Python function implementation, it may be default to center and standardize the variables). Since I've already told some students to report unstandardized results, I'll accept either implementation. For part 1, use either standardized or unstandardized variables. For part 2, make sure to discuss the difference in results between standardized and unstandardized variables for the last question.
